# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/pagination/fieldset-004-ref.html
# Commit: 0186165ee1d8
# Full Hash: 0186165ee1d84f56362ba16bbda4eefa1d47fd60
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-11-26 05:19:55
# Regressor Bug: 471015
# File Overlap Count: 1
# Description:
#   Bug 471015 - [css-break] Implement <fieldset> fragmentation.  r=TYLin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D53709
# ==============================================================================

diff -r 9cf1851c76dc -r 0186165ee1d8 layout/reftests/pagination/fieldset-004-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/reftests/pagination/fieldset-004-ref.html	Mon Nov 25 18:05:16 2019 +0000
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html class="reftest-paged">
+<head>
+  <title>Reference: fieldset fragmentation</title>
+  <link rel="author" title="Mats Palmgren" href="https://bugzilla.mozilla.org/show_bug.cgi?id=471015">
+  <meta name="flags" content="paged">
+  <style>
+@page { size:5in 3in; margin:0.5in; }
+html,body {
+    color:black; background-color:white; font:0.5in/1 monospace; padding:0; margin:0;
+}
+fieldset {
+  position: relative;
+  padding: 0;
+  margin: 0;
+  padding-top: 0.5em;
+  margin-top: 0.5em;
+  border: 1em solid black;
+  border-bottom-color: white;
+  page-break-after: always;
+}
+div {
+  position: absolute;
+  top: -1em;
+  width: 3em;
+  height: 1em;
+  background-color: white;
+}
+p { margin: 0; }
+x { display: block; height: 1em; }
+</style>
+</head>
+<body>
+  <fieldset><div></div><x>A<br>B</x></fieldset>
+  <p style="padding-left:1em">C</p><p style="margin-top:-1em">AFTER</p>
+</body>
+</html>